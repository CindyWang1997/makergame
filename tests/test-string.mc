int main () {
  string c;
  c = "hello world";
  printstr(c);
  printstr("goodbye world");
  printstr("this is \"right\"");
  printstr("Either\\or is okay");
  printstr("multiple\nlines");
  return 0;
}
