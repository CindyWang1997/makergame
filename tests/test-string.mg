object main {
event create {
  string c;
  c = "hello world";
  std::print::s(c);
  std::print::s("goodbye world");
  std::print::s("this is \"right\"");
  std::print::s("Either\\or is okay");
  std::print::s("multiple\nlines");
  std::end_game();
  return;
}
}
