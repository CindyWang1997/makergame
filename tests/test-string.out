hello world
goodbye world
this is "right"
Either\or is okay
multiple
lines
